marimo/_server/export/exporter.py

```diff
@@ -119,6 +120,25 @@ def export_as_script(
         download_filename = get_download_filename(file_manager, ".script.py")
         return code, download_filename
 
+    def export_as_ipynb(
+        self,
+        file_manager: AppFileManager,
+    ) -> tuple[str, str]:
+        import nbformat
+
+        notebook = nbformat.v4.new_notebook()  # type: ignore
+        graph = file_manager.app.graph
+        notebook["cells"] = [
```

Comment:
You could use `exporter/utils.is_markdown` to set markdown cells

Reply:
oh sweet, done -- thanks